src/main/java/org/folio/circulation/support/CirculationRulesClient.java

```diff
@@ -17,21 +17,21 @@
   private final OkapiHttpClient client;
   private final URL root;
 
-  CirculationRulesClient(OkapiHttpClient client, WebContext context)
+  CirculationRulesClient(OkapiHttpClient client, WebContext context, String policyPath)
     throws MalformedURLException {
 
     this.client = client;
-    root = context.getOkapiBasedUrl("/circulation/rules/loan-policy");
+    root = context.getOkapiBasedUrl(policyPath);
   }
 
   public void applyRules(
-    String loanTypeId,
+    String circulationTypeId,
```

Comment:
Can you help me understand why this has been renamed?

Reply:
Because this client is not only used for loans but also for requests as well, renaming it to make it more generic.

Reply:
I agree this is confusing now that the circulation rules are used for different kinds of policies.

My concern is that the parameter to the circulation rules is called `loan_type_id` and the properties (from the item) that are used to determine the value for this parameter are also called a `loan type`, and that this name does not fit with either of those.

Does that make sense?

Reply:
I see your point. I was thinking of incrementally updating this loan_type_id name as the opportunity arises.  However for now I reverted it to loanTypeId.